src/App.test.js

```diff
@@ -3,6 +3,8 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/Hex 2 RGB/i);
   expect(linkElement).toBeInTheDocument();
```

Comment:
Make sure you replace all linkElement with titleElement and continue giving meaningful names